book/modules/appendix2.tex

```diff
@@ -324,8 +417,38 @@
 
 	\begin{example}
 		Use row reduction to find the complete solution to $\systeme{x+y+z=1,y-z=2}$
-	
-		XXX Finish. Express answer both in vector form and as a set.
+		
+		The corresponding augmented matrix for the system is
+		\[
+			\begin{bmatrix}[ccc|c]
+			1 & 1 & 1 & 1\\
+			0 & 1 & -1 & 2
+			\end{bmatrix}
+		\]
+		
+		The matrix is already in pre-reduced row echelon form, so we only need to zero above each pivot.
+		\[
+			\begin{bmatrix}[ccc|c]
+			1 & 1 & 1 & 1\\
+			0 & 1 & -1 & 2
+			\end{bmatrix}
+			\xrightarrow{\text{row}_1\mapsto\text{row}_1-\text{row}_2}
+			\begin{bmatrix}[ccc|c]
+			1 & 0 & 2 & -1\\
+			0 & 1 & -1 & 2
+			\end{bmatrix}
+		\]
+		
+		The third column is a free variable column, so we introduce the arbitrary equation $z=t$ and solve the system in terms of $t$.
+		
+		The complete solution to the system in vector form is
```

Comment:
Keep the part you had before where you listed the system of equations with the `z=t` equation added in.